.changeset/new-mangos-study.md

```diff
@@ -0,0 +1,5 @@
+---
+"@10up/eslint-config": patch
```

Comment:
This should be a major as well.

Reply:
I didn’t think so. The only major is that it drops Node 12 in a nested dependency but we’re already enforcing that so I didn’t see the need

Reply:
Toolkit still supports Node 12 and eslint-config is a dependency that now will only support Node >= 16.

Reply:
Ahh! Missed that! Will update !

Reply:
@nicholasio this will create though a need to have a breaking version of Toolkit, are we ready for that?

Reply:
Yeah, I'll release the pending stuff in the current major then release this one. I'll also wait a bit before I get this one merged just in case there are other things we want to include in the current major.

Reply:
I was waiting for Linaria to get started on new Toolkit major but let me know if you want me to get started on a branch/PR for toolkit 5